doc/content/integrations/pubsub/mqtt-client/_index.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "MQTT Client"
+description: ""
+weight: 1
+---
+
+This section explains how to configure a Pub/Sub integration using the built-in MQTT client.
+
+<!--more-->
+
+Creating a Pub/Sub integration with the built in MQTT client requires you to have an MQTT server running with an available endpoint. [Mosquitto](https://mosquitto.org/) is a popular open source MQTT Server you may use, but {{% tts %}} MQTT client works with any MQTT server.
+
+> No API key is needed for Pub/Sub messaging. All messages received by the Application Server are assumed to be authorized, so be sure to configure appropriate security on your MQTT server.
+
+In your application select the **Pub/Subs** submenu from the **Integrations** side menu. Clicking on the **+ Add Pub/Sub** button will open the Pub/Sub creation screen.
+
+Give your Pub/Sub an **ID**, choose a **Pub/Sub format**, and enter a **Base topic** to subscribe or publish to.
+
+Select **MQTT** as provider.
+
+{{< figure src="mqtt.png" alt="Pub/Sub creation screen" >}}
+
+If using client certificates, enable **Secure**. You will be asked to upload the **Root CA certificate**, the **Client certificate**, and the **Client private key.**
+
+Enter the endpoint of the MQTT server in the **Server URL** field, including the protocol and port. 
+
```

Comment:
Here, I would point out that it is good to generally check the accessibility of the specified port on the given URL, before deploying the integration. Btw, is there a place a user can see all supported URL formats? I missed those in here.

Reply:
we should do a 'protocol' dropdown - making an issue for this